tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource

```diff
@@ -0,0 +1,74 @@
+*** Settings ***
+Documentation    Collection of keywords to interact with Data Connections
+Resource       ../../../../Page/Components/Components.resource
+Resource       ../../../../Common.robot
+Resource       Projects.resource
+
+
+*** Variables ***
+${S3_NAME_DC_INPUT_XP}=          xpath=//input[@aria-label="AWS field Name"]
+${S3_KEY_DC_INPUT_XP}=          xpath=//input[@aria-label="AWS field AWS_ACCESS_KEY_ID"]
+${S3_SECRET_DC_INPUT_XP}=          xpath=//input[@aria-label="AWS field AWS_SECRET_ACCESS_KEY"]
+${S3_ENDPOINT_DC_INPUT_XP}=          xpath=//input[@aria-label="AWS field AWS_S3_ENDPOINT"]
+${S3_REGION_DC_INPUT_XP}=          xpath=//input[@aria-label="AWS field AWS_DEFAULT_REGION"]
+${DC_SECTION_XP}=             xpath=//div[div//h4[@id="data-connections"]]
+${DC_ADD_BTN_1_XP}=           ${DC_SECTION_XP}/div/button[text()="Add data connection"]
+${DC_ADD_BTN_2_XP}=           xpath=//footer/button[text()="Add data connection"]
+
+
+*** Keywords ***
+Create S3 Data Connection
+    [Documentation]    Creates a S3 Data Connection from DS Project details page
+    [Arguments]    ${project_title}    ${dc_name}    ${aws_access_key}    ${aws_secret_access}    
+    ...            ${aws_s3_endpoint}    ${aws_region}    ${connected_workbench}=${NONE}    ${press_cancel}=${FALSE}
+    Click Button    ${DC_ADD_BTN_1_XP} 
+    Wait Until Page Contains Element    ${S3_NAME_DC_INPUT_XP}
+    Input Text    ${S3_NAME_DC_INPUT_XP}    ${dc_name}
+    Input Text    ${S3_KEY_DC_INPUT_XP}    ${aws_access_key}
+    Input Text    ${S3_SECRET_DC_INPUT_XP}    ${aws_secret_access}
+    Input Text    ${S3_ENDPOINT_DC_INPUT_XP}    ${aws_s3_endpoint}
+    Input Text    ${S3_REGION_DC_INPUT_XP}    ${aws_region}
+    IF    ${press_cancel} == ${TRUE}
+        Click Button    ${GENERIC_CANCEL_BTN_XP}
+    ELSE
+        Wait Until Element Is Enabled    ${STORAGE_ADD_BTN_2_XP}
+        Click Button    ${STORAGE_ADD_BTN_2_XP}
+    END
+    Wait Until Modal Disappears
+    Wait Until Project Is Open    project_title=${project_title}
+
+Data Connection Should Be Listed
+    [Documentation]    Checks a Data Connection is listed in DS Project details page
+    [Arguments]     ${name}   ${type}   ${connected_workbench}
```

Comment:
yeah, sorry for the mixed approach. I thought to spare character and improve readability/development to remove the prefix where not needed. So for example if a kw is called "Data Connection Is Listed" the ${name} argument cannot be anything else than th data connection name.

Where the prefix is added, like in "Create S3 Data connection" is to avoid ambiguity because there are many arguments. Same approach applied with storages for example. While workbench resource file was the first one created and I didn't follow the same rule. Changing it now it's very error-prone process IMO. 

If you all prefer me to change it, I can do in another PR if you don't mind since we need these tests for validating 1.19